integration/fuse/src/main/java/alluxio/fuse/AlluxioJniFuseFileSystem.java

```diff
@@ -466,14 +468,17 @@ private int releaseInternal(String path, FuseFileInfo fi) {
         }
       }
       if (ce != null) {
-        mCreateFileEntries.remove(ce);
         synchronized (ce) {
           ce.getOut().close();
         }
       }
     } catch (Throwable e) {
       LOG.error("Failed closing {}", path, e);
       return -ErrorCodes.EIO();
+    } finally {
+      if (ce != null) {
+        mCreateFileEntries.remove(ce);
```

Comment:
why we need to do it in finally?

Reply:
1. Previous logic remove ce before call `ce.getOut().close();`, I move it after this function to make sure it will be cleaned after the RPC return. So I can check if all close ops are returned based on the size of `mCreateFileEntries`
2. To respect previous logic: ce will be removed no matter `ce.getOut().close();` be executed successfully or not, I put it into finally block

Reply:
Got it, make sense!